tests/helpers/database.ts

```diff
@@ -0,0 +1,16 @@
+import { beforeEach, afterEach } from 'vitest';
+import { getPrisma } from '../../src/database';
+
+export function cleanDatabase() {
+  const clean = async () => {
+    const prisma = getPrisma();
+    await prisma.coreUnit.deleteMany();
+    await prisma.user.deleteMany();
```

Comment:
Nit: can you please generalise it in the next PR? Otherwise it's unclear that as soon as you create a new model, you also need to add line here. Eg https://github.com/prisma/docs/issues/451#issuecomment-1212842857